README.md

```diff
@@ -25,3 +25,12 @@ choose "Manage Plugin Repositories". Add https://plugins.jetbrains.com/plugins/b
 the list of repositories. After this is done, the bazel plugin compatible with recent IntelliJ
 versions should be available by clicking "Marketplace" in the Plugins and searching for Bazel.
 This has been tested with IntelliJ Idea Community 2022.1.2.
+
+## Credits
+Part of this code has been previously developed by the CTL team at [GS](https://gs.com). This
```

Comment:
This would need coordination with PR people. I think we should go with something short and sweet. "The code that this project is based on was written by Jonathan Perry at Goldman Sachs"

Reply:
Updated README file